quiver/benchmark_extraction.py

```diff
@@ -8,7 +8,7 @@
 import re
 from typing import Any, Dict, List, Union
 
-from .constants import *
+from quiver.constants import *
```

Comment:
Why?

Reply:
I thought that because `make benchmarks` is executed from the root of the project (and so is `quiver/benchmark_extraction.py`), this is necessary because `constants.py` wouldn't be found. I just learned that Python still finds it.

Addressed in [ced3251](https://github.com/OCR-D/quiver-back-end/pull/43/commits/ced3251ae1b711a1b197d8fc824536491ddf7cb9).